Description: glx: fix loading OpenGL library when libGL.so symlink is missing
Origin: upstream, https://code.qt.io/cgit/qt/qtbase.git/commit/?id=a35c6a49c938fefb
Last-Update: 2018-05-02

--- a/src/plugins/platforms/xcb/gl_integrations/xcb_glx/qglxintegration.cpp
+++ b/src/plugins/platforms/xcb/gl_integrations/xcb_glx/qglxintegration.cpp
@@ -585,6 +585,8 @@ QFunctionPointer QGLXContext::getProcAdd
                 extern const QString qt_gl_library_name();
 //                QLibrary lib(qt_gl_library_name());
                 QLibrary lib(QLatin1String("GL"));
+                if (!lib.load())
+                    lib.setFileNameAndVersion(QLatin1String("GL"), 1);
                 glXGetProcAddressARB = (qt_glXGetProcAddressARB) lib.resolve("glXGetProcAddressARB");
 #endif
             }
